packages/sdk-v2/src/sdk/base/sdk.ts

```diff
@@ -76,11 +76,13 @@ export class BaseSDK {
       signer
         .getAddress()
         .then(async () => void (this.signer = signer))
-        .catch(e => {
-          console.warn("BaseSDK: provider has no signer", { signer, provider, e });
+        .catch(() => {
+          // todo @l03tj3: revert back
```

Comment:
if the PR isn't a draft anymore, it shouldn't contain any TODOs (only if they're out of this PR's scope and you're going to open separate tasks in the icebox)

Reply:
done